portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -101,8 +101,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       // Get the message entered by the user.
       String message = request.getParameter("text-input");
 
-      //get number of comments
-      numberOfCommentsToDisplay = getNumberOfCommentsToDisplay(request);
+      //get sender
+      String sender = getParameter(request, "sender", "Steven");
+
+      //get type of comment
+      String commentType = getParameter(request, "tags", "Default")
```

Comment:
**super nit:** please leave a space between comment symbol and comment.
                Also please use capitalization and punctuation for comments